### PR TITLE
Add proposed authorship rules and info

### LINKED ISSUE #1: Decide on rules for authorship
For the conditions for the previous paper, see the v2.0 paper README: https://github.com/astropy/astropy-v2.0-paper#rules-for-authorship

### PR DESCRIPTION
**Pull request by adrn**
Please feel free to leave comments or suggestions on the rules for authorship!

### TODO
- [x] Update link to authorship Google form before merging (but after comment period)

Closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,34 +55,33 @@ be sent to crawfordsm@gmail.com.  -->
...
-Cruz (on behalf of the Astropy Coordination Committee) and Steve
-Crawford (on behalf of the Paper II coordination committee). -->
+We invite you to become a co-author if any of the following applies to you:
+
+   - You have an official role in the project, as defined on http://www.astropy.org/team.html
+   - You are a Voting Member, as listed on https://www.astropy.org/team.html#votingmembers
+   - You have contributed code to the core package
+   - You have contributed to Astropy Project infrastructure, including:
+      - Sphinx plugins
+      - Pytest plugins
```
Comment (pllim) [inline README.md:65]: Do we have to be specific here? Are you only alluding to the plugins that actually reside within `astropy` GitHub org?

File: README.md [hunk 2]
```diff
@@ -55,34 +55,33 @@ be sent to crawfordsm@gmail.com.  -->
...
+      - Package template
+      - Astropy Website
+      - Learn Astropy
+   - You have contributed to one of the following Astropy coordinated affiliated packages (see http://affiliated.astropy.org/):
```
Comment (pllim) [inline README.md:69]:
```suggestion
   - You have contributed to one of the following Astropy coordinated or affiliated packages (see http://affiliated.astropy.org/):
```

**Review by pllim — COMMENTED** [review]
Just minor comments above. LGTM otherwise. Thanks!

File: README.md (hunk 1 above)
Comment (adrn) [reply README.md:65]: Ah, yes - I'll add that clarification!

**Review by nmearl — APPROVED** [review]
This looks good. Did we decide if we wanted to keep the separate sections, or just list the authors in a single chunk?

**Pull request merged**